src/main/java/frc/robot/RobotContainer.java

```diff
@@ -30,21 +36,24 @@
  */
 public class RobotContainer {
   
-  // The robot's subsystems and commands are defined here...
+  //Example
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
-
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
 
+  //Subsystems
   private final Drive m_driveSubsystem = new Drive();
+  private final Pneumatics m_pnSubsystem = new Pneumatics();
 
+  //HID
   public Joystick m_driveJoystick;
```

Comment:
Be careful about creating identifiers and not instantiating them - they can lead to some real nasty null pointer bugs if left unchecked. If this code is WIP, then I'd recommend adding a TODO comment just so that you and other developers know that this source code is going to change

Reply:
The joysticks are instantiated on 103 & 104 in RobotContainer when the robot attempts to identify the correct ones. If it can't find the two that we're using then it will default to ports 0 & 1. Good idea, I'll change it for the future in case if for some reason we have a subsystem or something else that might run before configureJoysticks() is called.